src/cmds/scripts/win_postinstall.py

```diff
@@ -350,12 +401,13 @@ def main():
     global pbs_sbin
     username = None
     password = None
-    if len(sys.argv) < 2:
+    global installtype
+    if len(sys.argv) < 5:
```

Comment:
This will fail if user doesn't pass -t \<type\> argument (which is supported as per design, as default type is server)

Reply:
Yes, only username and password is mandatory, we are not to forcing to provide "type",
As you mentioned, "server" is default.  For this case this condition works fine.